evidence/dharjeezy/0002-dharjeezy-09-2024-retention-evidence.md

```diff
@@ -0,0 +1,64 @@
+# Evidence-0002: Retention at Rank I
+
+|                 |                                |
+| --------------- |--------------------------------|
+| **Report Date** | Date of submission (2024/10/1) |
+| **Submitted by**| Damilare Akinlose              |
+
+
+## Member details
+
+- Matrix username: `@dharjeezy/:matrix.org`
+- Polkadot address: `12GyGD3QhT4i2JJpNzvMf96sxxBLWymz4RdGCxRH5Rj5agKW`
+- Current rank: `1`
+- Date of initial induction: `2024-04-29`
+- Date of last report: `2024-07-01`
+- Area(s) of Expertise/Interest: `FRAME & Runtime`
+
+
+## Reporting period
+
+- Start date: 2024/07/29
+- End date: 2024/10/29
+
+
+## Evidence
+I discussed about how to ensure that pallet MQ message processing is transactional and I also implemented the enforcement of transactional processing to [Pallet MQ message processing](https://github.com/paritytech/polkadot-sdk/pull/5198).
+
+I also had to refractor what I did before when I introduced a new extrinsic for full unbonding in pallet staking, but it was later advised that instead of introducing a new extrinsic, it will make sense
```

Comment:
```suggestion
I also had to refactor what I did before when I introduced a new extrinsic for full unbonding in pallet staking, but it was later advised that instead of introducing a new extrinsic, it will make sense
```